md/set-up-continuous-integration.md

```diff
@@ -5,7 +5,9 @@
 :::
 
 ::: warning
-**Note:** [Bonita Continuous Delivery add-on](https://documentation.bonitasoft.com/bcd/2.0/) comes with an out-of-the-box solution based on Jenkins CI. 
+**Note:**
+* The solution described here relies on the BonitaStudioBuilder which is deprecated as of Bonita 7.7
```

Comment:
```suggestion
* The solution described here relies on the BonitaStudioBuilder, which has been deprecated since Bonita 7.7.0. Instead, we strongly encourage you to use the LA builder included in the tooling suite of Bonita Continuous Delivery add-on. One added-value is that LA builder does not need a studio to be installed.
```